src/App.tsx

```diff
@@ -10,22 +10,34 @@ import { MyTimer } from './components/Timer';
 import { usePrevious } from './hooks/usePrevious';
 import MyButton from './components/MyButton';
 import { useMyCallback } from './hooks/useMyCallback';
+import UserData from './mock/userdata';
+import { User } from './models/user';
+import MyList from './components/MyList/MyList';
 
 function App() {
   const [showTime, setShowTime] = useState(true);
   const previousShowTime = usePrevious(showTime);
   const btnContent = useMemo(() => <>show time</>, []);
```

Comment:
Isn't related to this PR, but I don't think you need to remember this text

Reply:
If it's just a string I could pass that as a prop and not worry about it, but since it's a fragment passed as a children it's necessary to memoize it to prevent unnecessary renders